xds/src/test/java/io/grpc/xds/EnvoyServerProtoDataTest.java

```diff
@@ -147,7 +119,8 @@ private static FilterChain createInFilter() {
                     .addApplicationProtocols("managed-mtls")
                     .build())
             .setTransportSocket(TransportSocket.newBuilder().setName("tls")
-                .setTypedConfig(Any.pack(CommonTlsContextTestsUtil.buildTestDownstreamTlsContext()))
+                .setTypedConfig(
+                    Any.pack(CommonTlsContextTestsUtil.buildTestDownstreamTlsContextV2()))
```

Comment:
Why `V2` ? With all Listener related stuff migrating to v3 why can't this be v3 as well?

Reply:
Good catch. I missed that.